examples/cuda/scripts/export.py

```diff
@@ -0,0 +1,115 @@
+# Copyright © 2023 Apple Inc. All rights reserved.
+#
+# Please refer to the license found in the LICENSE file in the root directory of the source tree.
+
+import argparse
+import pathlib
+
+import torch
+
+# pyre-fixme[21]: Could not find module `executorch.backends.apple.coreml.compiler`.
+from executorch.backends.cuda.cuda_backend import CudaBackend
+
+# pyre-fixme[21]: Could not find module `executorch.backends.apple.coreml.partition`.
+from executorch.backends.cuda.cuda_partitioner import CudaPartitioner
+
+from executorch.examples.models import MODEL_NAME_TO_MODEL
+from executorch.examples.models.model_factory import EagerModelFactory
+
+from executorch.exir import EdgeCompileConfig, to_edge_transform_and_lower
+
+from executorch.extension.export_util.utils import save_pte_program
+from torch._inductor.decomposition import conv1d_to_conv2d
+from torch.nn.attention import SDPBackend
+
+# Script to export a model with coreml delegation.
+
+_EDGE_COMPILE_CONFIG = EdgeCompileConfig(
+    _check_ir_validity=False,
+    _skip_dim_order=True,  # TODO(T182928844): enable dim_order in backend
```

Comment:
we should turn it on now.

Reply:
I'm too afraid to turn it on in this PR lol